apps/web/src/config/index.ts

```diff
@@ -68,3 +68,13 @@ export const FEATURE_FLAGS = Object.values(FeatureFlagsKeysEnum).reduce((acc, ke
 export const HUBSPOT_PORTAL_ID = window._env_.REACT_APP_HUBSPOT_EMBED || process.env.REACT_APP_HUBSPOT_EMBED;
 
 export const IS_EU_ENV = (ENV === 'production' || ENV === 'prod') && API_ROOT.includes('eu.api.novu.co');
+
+export const IS_CLERK_AUTH_ENABLED =
+  window._env_.REACT_APP_IS_CLERK_AUTH_ENABLED === 'true' || process.env.REACT_APP_IS_CLERK_AUTH_ENABLED === 'true';
+
+export const CLERK_PUBLISHABLE_KEY =
+  window._env_.REACT_APP_CLERK_PUBLISHABLE_KEY || process.env.REACT_APP_CLERK_PUBLISHABLE_KEY;
+
+if (!CLERK_PUBLISHABLE_KEY) {
+  throw new Error('Missing Clerk Publishable Key');
```

Comment:
💯

Reply:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** How will this affect self-hosted users?‏